metrics-httpclient/src/main/java/com/codahale/metrics/httpclient/InstrumentedHttpClientConnectionManager.java

```diff
@@ -30,80 +30,48 @@
 
     protected static Registry<ConnectionSocketFactory> getDefaultRegistry() {
         return RegistryBuilder.<ConnectionSocketFactory>create()
-            .register("http", PlainConnectionSocketFactory.getSocketFactory())
-            .register("https", SSLConnectionSocketFactory.getSocketFactory())
-            .build();
+                .register("http", PlainConnectionSocketFactory.getSocketFactory())
+                .register("https", SSLConnectionSocketFactory.getSocketFactory())
+                .build();
     }
 
     private final MetricRegistry metricsRegistry;
     private final String name;
 
-    public InstrumentedHttpClientConnectionManager(MetricRegistry metricRegistry) {
```

Comment:
We try to maintain backward compatibility with the previous version of Metrics. Please don't remove the public constructors, otherwise anybody who uses them will be forced to migrate their code to `Builder`, which should be optional.

Reply:
Arteam, my understanding was that this was just added to the development branch and did not yet made it to any stable release. In this case it is probably ok to make backwards-incompatible changes? Or do you still think that I should add these constructors back?

Reply:
Please add them back. Yes, it's a development branch, but we still don't want to break the compatibility across major updates.